packages/appstore/src/components/product-card/product-card.scss

```
@@ -0,0 +1,135 @@
+.product-card {
+    width: 200px;
+    height: 120px;
+    border-radius: 8px;
+    overflow: hidden;
+
+    @include mobile {
+        width: 144px;
+        height: 96px;
+    }
+
+    &__trade-type {
+        position: relative;
+
+        &--body {
+            padding: 16px;
```

```suggestion
            padding: 1.6rem;
```
Please use `rem` instead of `px` for sizing

---

fixed in current commit..